components/builder-web/app/actions/feature-flags.ts

```diff
@@ -1,72 +0,0 @@
-// Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
```

Comment:
I like that we are loading feature flags from the server, but won't there likely be cases where we will want to feature flag things on the UI end as well? Seems like it would be nice to have server flags which can be overridden by the UI / cookies.

Reply:
Possibly, but we don't have a need for such a thing today, and it makes sense to have features be toggleable in one place using the means we've already created. We can always pull this back in when we need more than that'll provide for us.